### PR TITLE
feat: QuerySet filtration for Form-Assesment API

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/assesment/views.py [hunk 1]
```diff
@@ -92,9 +92,17 @@ class QuestionViewSet(viewsets.ModelViewSet):
...
 class FormAssesmentViewSet(viewsets.ModelViewSet):
-    queryset = FormAssesment.objects.all()
     serializer_class = FormAssesmentSerializer
-    permission_classes = [permissions.IsAuthenticated,]
+    permission_classes = [IsPatient]
+
+    def get_queryset(self):
+        if self.request.user.role == Roles.ADMIN:
+            return FormAssesment.objects.all()
+        if self.request.user.role == Roles.DOCTOR:
```
Comment (Raam124) [inline apps/assesment/views.py:101]: doctor will be able to see all the footprint to pick one to himself initially that feature is missing here

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/assesment/views.py:102]: Change made

File: apps/assesment/views.py [hunk 2]
```diff
@@ -92,9 +93,17 @@ class QuestionViewSet(viewsets.ModelViewSet):
...
+            return FormAssesment.objects.filter(Q(doctor=self.request.user.doctor) | Q(doctor__isnull=True))
```
Comment (Raam124) [inline apps/assesment/views.py:103]: did you manually check is this `doctor__isnull=True` condition working? if yes attach a screenshot here of the result

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/views.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/assesment/views.py:103]: @Raam124 
image

**Review by Raam124 — APPROVED** [review]

**Pull request merged**